package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "nokta.dz",
+  "name": "nukta.dz",
   "version": "0.1.0",
   "description": "A collection of DZ jokes",
```

Comment:
in now a days nothing less original then attaching dz to a thing haha, but yeah joke.dz is good a so, it will fit !